pyproject.toml

```diff
@@ -9,10 +9,7 @@ packages = [
 version = "0.1.2"
 
 [tool.poetry.dependencies]
-# common-model = { git = "https://github.com/kvh/common-model.git" }
-# datacopy = { git = "https://github.com/kvh/dcp.git" }
-common-model = "^0.2.2"
-datacopy = "^0.1.15"
+pandas = "^1"

```

Comment:
We don't actually use pandas, but `DataFrame`s are in a couple of type annotations for port methods.

Reply:
Yeah, we could stringify those type annotations? Then up to user to install to get hints

Reply:
Like is `-> "pandas.DataFrame"` allowed?

Reply:
I'd say more important for type hints to work than minimizing dependencies (any data scientist gonna have pandas installed anyways)

Reply:
Stringifying annotations doesn't work here, but could do a conditional import instead if we wanted to remove the pandas dep. I'd lean towards leaving it, since you probably need it to actually write any node code.

Reply:
👍